brainsets/utils/openneuro/__init__.py

```
@@ -0,0 +1,33 @@
+    "OPENNEURO_S3_BUCKET",
+    "construct_s3_url_from_path",
+    "download_dataset_description",
+    "download_recording",
+    "fetch_all_filenames",
+    "fetch_participants_tsv",
+    "validate_dataset_id",
+    "validate_dataset_version",
+    "OpenNeuroPipeline",
+    "OpenNeuroEEGPipeline",
+    "OpenNeuroIEEGPipeline",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)